dist/api_err.py

```diff
@@ -83,19 +83,30 @@ def __init__(self, name, value, desc, long_desc=None, **flags):
         setting.'''),
 ]
 
-# Update the #defines in the wiredtiger.in file.
-tmp_file = '__tmp_api_err' + str(os.getpid())
-tfile = open(tmp_file, 'w')
-skip = 0
-for line in open('../src/include/wiredtiger.in', 'r'):
-    if not skip:
-        tfile.write(line)
-    if line.count('Error return section: END'):
+sub_errors = [
+    Error('WT_NONE', -32000,
+    'last API call was successful', '''
+    This is the default sub-level error code that should be used when there is no
+    sub-level error to pair with an error. It indicates that no further context
+    exists or is necessary.'''),
+    Error('WT_COMPACTION_ALREADY_RUNNING', -32001,
+    "Cannot reconfigure background compaction while it's already running", '''
+    This error is generated when the user tries to reconfigure the background
+    compaction while it is already running.'''),
+    Error('WT_SESSION_MAX', -32002,
+    "out of sessions, configured for XXX (including internal sessions)", '''
+    This sub-level error is generated when the user has created the max amount of
+    sessions configured.'''),
+]
+
+# Checks if return section begins and updates error defines
```

Comment:
Please can you move the functions in this file to be before any code that is not in functions - this will make the code easier to read.

Reply:
Fixed.